### DISCUSSION TITLE
Unique Selling Points

### DISCUSSION THREAD

**Discussion by bobturneruk** [pykale/pykale#211] [General] [answered]
What are the most important things about `PyKale` that differentiate it from other similar projects e.g.

* Easy to install.
* Example notebooks.
* Easy to use.
* Can use without coding (just with YAML #163).

**Comment by bobturneruk** [comment]
- Easier to make outputs reproducible

**Comment by bobturneruk** [comment]
Not necessarily unique, but important:
- Multi source
- Domain adaptation

**Comment by haipinglu** [comment]
Design based on five years of teaching machine learning to about ~1000 students of diverse background.

**Comment by haipinglu** [comment, marked as answer]
Design based on interdisciplinary research collaboration over the past four years.
